.github/workflows/windows.yml

```
@@ -17,32 +14,41 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+<<<<<<< HEAD
   mingw:
     runs-on: windows-2022
     strategy:
       matrix:
         architecture: [x64, x86]
 
+=======
+  # 1) MSVC on windows-latest
+  msvc-latest:
+    runs-on: windows-latest
+>>>>>>> 2025ebb5 (ci(windows): re-enable full Windows matrix)
     steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - uses: actions/checkout@v3
+      - name: Build with MSVC
+        run: cmake --build build --config Release
+
+  # 2) MinGW on windows-latest
+  mingw-latest:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
       - name: Set up MinGW
-        uses: egor-tensin/setup-mingw@84c781b557efd538dec66bde06988d81cd3138cf # v2.2.0
-        with:
-          platform: ${{ matrix.architecture }}
-          version: 12.2.0 # https://github.com/egor-tensin/setup-mingw/issues/14
-      - name: Run CMake
-        run: cmake -S . -B build -G "MinGW Makefiles" -DCMAKE_BUILD_TYPE=Debug -DJSON_BuildTests=On
-      - name: Build
-        run: cmake --build build --parallel 10
-      - name: Test
-        run: cd build ; ctest -j 10 -C Debug --output-on-failure
+        uses: egor-tensin/setup-mingw@v2.2.0
```

Please keep the hashes like in the original version.

```suggestion
        uses: egor-tensin/setup-mingw@84c781b557efd538dec66bde06988d81cd3138cf # v2.2.0
```